stories/FlatBookCover.stories.js

```diff
@@ -0,0 +1,35 @@
+import FlatBookCover from '../openlibrary/components/LibraryExplorer/components/FlatBookCover.vue';
+
+export default {
+  title: 'Explorer/FlatBookCover',
+  component: FlatBookCover
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { FlatBookCover },
+  template:'<flat-book-cover :book="book" />',
```

Comment:
Can we use Pascal case? That's what I've been using in Vue land
```suggestion
  template:'<FlatBookCover :book="book" />',
```

Reply:
I was curious about the use of camel case in the project. We've been using kebab case in Wikimedia projects  after weighing up eh trade offs [1, 2]

("Unfortunately, due to HTML’s case insensitivity, DOM templates must still use kebab-case."). Also note issues on case insensitive 

Might I suggest you add eslint supprot to enforce this if you've settled on that convention.
https://eslint.vuejs.org/rules/component-definition-name-casing.html

[1] https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended
[2] https://vuejs.org/v2/guide/components-props.html#Prop-Casing-camelCase-vs-kebab-case